docs/pages/en-us/Getting-Started.md

```diff
@@ -539,33 +327,35 @@ In the `ConfigurationSource` above, we used JSON-encoded files with the followin
 - `CAF_CONFIG_PATH: /etc/sampleapp/config`
 - `CAF_APPNAME: sampleappgroup/sampleapp`
 
-Given this configuration, you would configure Auditing by creating a file named `cfg_sampleappgroup_sampleapp_KafkaAuditConfiguration` in the `/etc/sampleapp/config/` directory. The contents of this file should be similar to the following:
+Given this configuration, you would configure Auditing by creating a file named `cfg_sampleappgroup_sampleapp_ElasticsearchAuditConfiguration` in the `/etc/sampleapp/config/` directory. The contents of this file should be similar to the following:
 
 	{
-	    "bootstrapServers": "<kafka broker>:<port number>",
-	    "acks": "all",
-	    "retries": "0"
+	    "hostAndPortValues": "<Elasticsearch_Cluser_Node_1>:<Port_Number>,<Elasticsearch_Cluser_Node_2>:<Port_Number>",
```

Comment:
3.0.0 will be released as hostAndPortValues. Given the current snapshot, during my manual test I will need to change this